crates/ruff/src/args.rs

```diff
@@ -527,6 +527,10 @@ pub struct FormatCommand {
     /// The option can only be used when formatting a single file. Range formatting of notebooks is unsupported.
     #[clap(long, help_heading = "Editor options", verbatim_doc_comment)]
     pub range: Option<FormatRange>,
+
+    /// Exit with a non-zero status code if any files were modified via format, even if all files were formatted successfully.
+    #[arg(long, help_heading = "Miscellaneous")]
+    pub exit_non_zero_on_format: bool,
```

Comment:
Do you find `ruff format --exit-non-zero-on-format` awkward? Should it be `ruff format --exit-non-zero-on-change` or `--exit-non-zero-on-write`? It might be nice to switch to an agnostic name.

Should we retain the `on_fix` language we use for `ruff check`? I see the motivation for not, but at the very least we probably want an alias for parity with `ruff check`?

Reply:
The current name was meant to "reflect" "exit non zero on (action)", to mirror "on fix".

As for the ergonomics? I really don't have an opinion. I can't imagine _anyone_ using this flag outside of some kind of config file or script file of some sort. In which case the name is really only as important as "can be discovered" and "name reflects what happens". So I think the bike shed on the name (past probably "exit non zero on ...") doesn't bother me. I'm happy to use whatever _yall_ like most!